src/comparisons.jl

```diff
@@ -1,7 +1,40 @@
-# This file is inspired from Polyhedra.jl
+# Some functions in this file are inspired from Polyhedra.jl
+#
+export set_rtol
```

Comment:
Any reason why `set_ztol` and `set_atol` are not exported?

Reply:
No; let's export those as well 👍